mobile/library/cc/request_headers_builder.h

```diff
@@ -11,6 +11,12 @@
 namespace Envoy {
 namespace Platform {
 
+// Available algorithms to compress requests.
+enum CompressionAlgorithm {
+  gzip,
```

Comment:
Should these follow the Envoy style guide recommendation:

Enum values using PascalCase (e.g., RoundRobin).